addons/stock/models/stock_lot.py

```diff
@@ -48,7 +48,7 @@ def _read_group_location_id(self, locations, domain):
         required=True, check_company=True)
     product_uom_id = fields.Many2one(
         'uom.uom', 'Unit of Measure',
-        related='product_id.uom_id', store=True)
+        related='product_id.uom_id')
```

Comment:
Shouldn't this one be kept, for tracking purpose?

Reply:
What do you mean ? If the source change, `product_uom_id` will be updated too. And both fields doesn't used tracking anyway (and even if they do, we don't want to repeat the information on every lots).